pkg/kubehound/graph/edge/role_bind_crb_cr_r.go

```diff
@@ -167,12 +170,47 @@ func (e *RoleBindCrbCrR) Stream(ctx context.Context, store storedb.Provider, c c
 			},
 		},
 		// Looking for all permissionSets link to the same namespace
+		// {
```

Comment:
Does we want to keep that comment? Why?

Reply:
This is a duplicate of what is below. We only need of them.